src/components/Pagination.vue

```diff
@@ -78,10 +78,24 @@ export default {
   methods: {
     nextPage() {
       const routeId = this.$route.params.id++;
-      return this.$router.push({ params: routeId  });    
+      const objectId = this.$route.params.id;
+      const pageNum = JSON.parse(objectId);
+      
+      if (pageNum === 8) {
+        return this.$router.push('/results');
+
+      }
```

Comment:
```suggestion      
      if (this.$route.params.id === '8') {
        return this.$router.push('/results');
      }
```

You can compare it as a string if you want, alternatively you could "throw" the id from a string to a number with `+this.$route.params.id` which would convert it into a number:

```suggestion
      if (+this.$route.params.id === 8) {
        return this.$router.push('/results');
      }
```

My recommendation would be for the former